File: tools/dbench/answers/go_sonnet/s1/e3_validate.go
package main

import "fmt"

func main() {
	ages := []int{34, -2, 19, 150, 42, 27}
	valid := 0
	invalid := 0
	sum := 0
	for _, age := range ages {
		if age >= 0 && age <= 120 {
			valid++
			sum += age
		} else {
			invalid++
		}
	}
	fmt.Println(valid)
	fmt.Println(invalid)
	fmt.Println(sum)
}

File: tools/dbench/answers/python_haiku/s2/e3_validate.py
ages = [34, -2, 19, 150, 42, 27]
valid = [a for a in ages if 0 <= a <= 120]
invalid = [a for a in ages if a < 0 or a > 120]
print(len(valid))
print(len(invalid))
print(sum(valid))
